diagnostics.tf

```diff
@@ -1,7 +1,6 @@
 locals {
   diagnostics_team = [
-    "Karsten1987",
-    "norro",
```

Comment:
@norro, you're on this list due to very early activity in the diagnostics release repository for ROS 2. Is it alright if you're removed from this team now?

Conversely @ralph-lange is there a separate justification you have for removing `norro` rather than just substituting yourself for Karsten?

Reply:
@Karsten1987 and @norro implemented the ROS 2 port of diagnostics together during their employment at Bosch. Since both have left the company, I would continue the maintenance of ROS 2 diagnostics.